daisy_workflows/image_import/import_image.sh

```diff
@@ -15,13 +15,17 @@
 
 set -x
 
+BYTES_1GB=1073741824
 URL="http://metadata/computeMetadata/v1/instance"
 DAISY_SOURCE_URL="$(curl -f -H Metadata-Flavor:Google ${URL}/attributes/daisy-sources-path)"
 SOURCE_DISK_FILE="$(curl -f -H Metadata-Flavor:Google ${URL}/attributes/source_disk_file)"
 SOURCEURL=${SOURCE_DISK_FILE}
```

Comment:
makes SOURCE_DISK_FILE unused; consider merging with above line so that SOURCEURL=$(curl...)

Reply:
sounds good to me!